README.md

```diff
@@ -141,6 +156,8 @@ file:
 
 ### DK
 
+*Attribution*: [The Danish Agency for Data Supply and Efficiency](http://sdfe.dk).
```

Comment:
"Agency for Data Supply and Efficiency" (without "The Danish" - here and numerous other places)